packages/dina-ui/components/seqdb/pcr-workflow/SangerPcrReactionStep.tsx

```diff
@@ -72,12 +78,33 @@ export function SangerPcrReactionStep({
       }
     }
 
+    if (performComplete) {
+      await save<PcrBatch>([
+        {
+          resource: {
+            id: pcrBatchId,
+            isCompleted: true,
+            type: "pcr-batch"
+          } as any,
+          type: "pcr-batch"
+        }
+      ], {
+        apiBaseUrl: "/seqdb-api"
+      });
+    }
+
     // Leave edit mode...
     if (!!setPerformSave) {
       setPerformSave(false);
     }
+
+    if (!!setPerformComplete){
+      setPerformComplete(false);
+    }
```

Comment:
This should be removed since we do not want to alter the existing state to false if just doing a normal save.